src/documentation/pages/components/drawer/examples/_close-button.vue

```diff
@@ -0,0 +1,219 @@
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- JavaScript -->
+<script>chi.drawer(document.getElementById('drawer-trigger-15'));<\/script>`
+            }
+          }
+        },
+        {
+          id: 'non-closable',
+          label: 'Non closable',
+          codeSnippets: {
+            webComponent: {
+              code: `<!-- Trigger -->
+<chi-button id="drawer-trigger-16" variant="flat" type="icon">
+  <chi-icon icon="menu" size="sm--2"></chi-icon>
+</chi-button>
+
+<!-- Drawer -->
+<chi-drawer id="interactive-drawer-16" position="left" backdrop non-closable no-header>
+  <div class="-p--2 -pt--6 -text">Drawer content here</div>
+</chi-drawer>
+
+<!-- JavaScript -->
+<script>
+  var drawer = document.getElementById('interactive-drawer-16');
+  var drawerTrigger = document.getElementById('drawer-trigger-16');
+  drawerTrigger.addEventListener('click', function(){
+    drawer.toggle();
+    // or drawer.active = !drawer.active;
+  });
+<\/script>`,
+              description: `Use the <code>non-closable</code> attribute to render Drawer without a close button`
+            },
+            vue: {
+              code: `<!-- Trigger -->
+<button class="chi-button -flat -icon" @click="() => toggleDrawer()">
+  <i class="chi-icon -sm--2 icon-menu" aria-hidden="true"></i>
+</button>
+
+<!-- Vue component -->
+<ChiDrawer
+  position="left"
+  :active="this.drawerActive"
+  :nonClosable="true"
+  :noHeader="true"
+  backdrop
+>
+  <div class="-p--2 -pt--6 -text">Drawer content here</div>
+</ChiDrawer>
+
+<!-- Data and Methods -->
+data: {
+  drawerActive : false;
+},
+methods: {
+  toggleDrawer() {
+    this.drawerActive = !this.drawerActive;
+  }
+}`,
+              description: `Use the prop <code>nonClosable</code> to render Drawer without a close button`
+            },
+            htmlBlueprint: {
+              code: `<!-- Trigger -->
+<button id="drawer-trigger-17" class="chi-button -flat -icon chi-drawer__trigger" data-target="#drawer-17" aria-label="Toggle navigation">
+  <div class="chi-button__content">
+    <i class="chi-icon -sm--2 icon-menu" aria-hidden="true"></i>
+  </div>
+</button>
+
+<!-- Drawer -->
+<div class="chi-backdrop -closed">
+  <div class="chi-backdrop__wrapper">
+    <div id="drawer-17" class="chi-drawer -left">
+      <div class="chi-drawer__content">
+        <div class="-p--2 -pt--6 -text">Drawer content here</div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- JavaScript -->
+<script>chi.drawer(document.getElementById('drawer-trigger-17'));<\/script>`
+            }
+          }
+        }
+      ],
+      exampleTabs: [
+        {
+          active: true,
+          id: 'webcomponent',
+          label: 'Web component'
+        },
+        {
+          id: 'vue',
+          label: 'Vue'
+        },
+        {
+          id: 'htmlblueprint',
+          label: 'HTML blueprint'
+        }
+      ],
+      exampleId: 'close-button',
+      closable: false
+    };
```

Comment:
@josecarlosgonzalezv Trigger codesnippets for webcomponent and vue are missing for closable example.
<img width="514" alt="codeSnippet" src="https://user-images.githubusercontent.com/92325462/160314169-d75cc8f8-d9dd-4628-bded-a6d2e75c122a.png">

Reply:
By the way, I have checked all the other examples, nice catch!